### DISCUSSION TITLE
Is it possible to install MySQL 5.7.39 although 5.7.38 is the newest in brew?

### DISCUSSION THREAD

**Discussion by petersmoeller** [Homebrew/discussions#3519] [Casks]
Hi there, 
MySQL 5.7.39 has built in support for arm SoCs, so I would guess that performance might be improved a bit on M1 macs? 

5.7.38 is the latest on homebrew though. Is it possible to install 5.7.39 anyway? Maybe a special flag/dev/head sort of thing? 

Regards, 

Peter

**Comment by SMillerDev** [comment]
You can compile it yourself, then you can have any version you want

**Reply by petersmoeller** [reply]
Thanks, 
Was really hoping for an easy way to do that through homebrew - doesn't seem to be that easy unfortunately.

**Reply by SMillerDev** [reply]
You can always follow https://docs.brew.sh/Updating-Software-in-Homebrew
